model/atmosphere/dycore/README.md

```diff
@@ -4,6 +4,66 @@
 
 Contains code ported from ICON `src/atm_dyn_iconam`, which is the dynamical core of the ICON model.
 
+## Attention
```

Comment:
I would rename this to something like *Treatment of variables with different vertical levels*

Reply:
Thanks for the good name. I will rename it.